app/(gcforms)/[locale]/layout.tsx

```diff
@@ -12,6 +12,7 @@ export default async function Layout({ children }: { children: React.ReactNode }
     FeatureFlags.saveAndResume,
     FeatureFlags.formTimer,
     FeatureFlags.hCaptcha,
+    FeatureFlags.caretakerPeriod,
```

Comment:
We should just call this banner or something as it will change depending on what the text is

Reply:
IMO: that seems dangerous to me since what banner content would shown is not clear. The content strings can be changed and what banner is shown is unclear IMO.

Reply:
For now let's just leave it and come back to it later to get this out the door